Marlin/src/lcd/dogm/ultralcd_DOGM.cpp

```diff
@@ -220,7 +220,12 @@ void MarlinUI::set_font(const MarlinFont font_nr) {
 void MarlinUI::init_lcd() {
 
   #if PIN_EXISTS(LCD_BACKLIGHT) // Enable LCD backlight
-    OUT_WRITE(LCD_BACKLIGHT_PIN, HIGH);
+    #if ENABLED(MKS_ROBIN_TFT)
```

Comment:
It's preferable to have a flag set in `Conditionals_LCD.h` that is set based on `MKS_ROBIN_TFT` (and future displays) rather than add conditions specific to a particular LCD in this file.